utils/chcache/README.md

```diff
@@ -0,0 +1,23 @@
+## chcache
```

Comment:
In the short term we might be ok having a different folder and downloading deps, but if we want to make this official then we should integrate with the `rust/` folder and vendor in the dependencies so no downloads are necessary.